app/models/further_information_request_item.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: further_information_request_items
+#
+#  id                             :bigint           not null, primary key
+#  assessor_notes                 :text
+#  information_type               :string
+#  response                       :text
+#  created_at                     :datetime         not null
+#  updated_at                     :datetime         not null
+#  further_information_request_id :bigint
+#
+# Indexes
+#
+#  index_fi_request_items_on_fi_request_id  (further_information_request_id)
+#
+class FurtherInformationRequestItem < ApplicationRecord
+  belongs_to :further_information_request, inverse_of: :items
+  has_many :documents, as: :documentable
```

Comment:
```suggestion
  has_one :document, as: :documentable
```

I suspect we'll end up only needing one document per request item (since a document can have many uploads/translations) but not a major issue - we can see how we use it.

Reply:
Oh yeah. Meant to do that 🤦